storage/container_storage_interface/persistent-storage-csi-vsphere.adoc

```diff
@@ -16,7 +16,7 @@ To create CSI-provisioned persistent volumes (PVs) that mount to vSphere storage
 
 * *vSphere CSI Driver Operator*: The Operator provides a storage class, called `thin-csi`, that you can use to create persistent volumes claims (PVCs). The vSphere CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand, eliminating the need for cluster administrators to pre-provision storage.
 
-* *vSphere CSI driver*: The driver enables you to create and mount vSphere PVs. In {product-title} 4.12, the driver version is 2.6.1. The vSphere CSI driver supports all of the file systems supported by the underlying Red Hat Core OS release, including XFS and Ext4. For more information about supported file systems, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/managing_file_systems/assembly_overview-of-available-file-systems_managing-file-systems[Overview of available file systems].
+* *vSphere CSI driver*: The driver enables you to create and mount vSphere PVs. In {product-title} 4.12.20, and later, the driver version is 2.7.1. The version is 2.6.1 in {product-title} 4.12 versions earlier than 4.12.20. The vSphere CSI driver supports all of the file systems supported by the underlying Red Hat Core OS release, including XFS and Ext4. For more information about supported file systems, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/managing_file_systems/overview-of-available-file-systems_managing-file-systems[Overview of available file systems].
```

Comment:
Is 4.12 running rhel 9 ? If not I don't think we should update that reference.

https://docs.openshift.com/container-platform/4.12/release_notes/ocp-4-12-release-notes.html#ocp-4-12-rhcos-rhel-8-6-packages

Reply:
Fixed to ref RHEL 8.